src/app/proposals/shared/supportedABIs.ts

```diff
@@ -1,32 +1,18 @@
 import { DAOTreasuryAbi } from '@/lib/abis/DAOTreasuryAbi'
 import { RIFTokenAbi } from '@/lib/abis/RIFTokenAbi'
-import { SimplifiedRewardDistributorAbi } from '@/lib/abis/SimplifiedRewardDistributorAbi'
-import { BuilderRegistryAbi } from '@/lib/abis/v2/BuilderRegistryAbi'
+import { BuilderRegistryAbi } from '@/lib/abis/collective-rewards/BuilderRegistryAbi'
 import { HTMLProps } from 'react'
 import { AbiFunction, AbiParameterToPrimitiveType } from 'viem'
 
-export const abiNames = [
-  'DAOTreasuryAbi',
-  'RIFTokenAbi',
-  'BuilderRegistryAbi',
-  // CR MVP: To keep compatibility with the MVP logs
-  'SimplifiedRewardDistributorAbi',
-] as const
-export const abis = [
-  DAOTreasuryAbi,
-  RIFTokenAbi,
-  BuilderRegistryAbi,
-  // CR MVP: To keep compatibility with the MVP logs
```

Comment:
We cannot remove this, if we do it, the actions from the MVP proposal will be empty like this [one](https://qa.cr.rootstockcollective.xyz/proposals/91633001490326175140258367116357841016460732489920126069844370060726414061262)